main-app/app/scripts/services/directives.js

```diff
@@ -1,20 +1,22 @@
-(function(){
-//Todo: Square Selection Directive
-angular.module('noughtsAndCrossesApp')
+/*(function(){*/
+
+angular.module('tombola.noughtsAndCrossesApp.directives.gameSquare')
```

Comment:
This should be just 'tombola.noughtsAndCrossesApp.directives' - the directive itself is gameSquare